extension-manifest-v3/scripts/build.js

```diff
@@ -136,7 +136,7 @@ if (manifest.declarative_net_request?.rule_resources) {
 }
 
 // generate license file
-import('../../tools/licenses.js');
+execSync('node ../tools/licenses.js', { stdio: 'inherit' });
```

Comment:
We need to be sure that this process ends before rollup starts, as it creates `node_modules` path inside of the v10 folder on the fly, and clears it out at the end.

Reply:
- [ ]